.devops/code-review-pipelines.yml

```
@@ -12,7 +12,7 @@ resources:
     - repository: pagopaEcommerceLocal
       type: github
       name: pagopa/pagopa-ecommerce-local 
-      ref: main
+      ref: CHK-2684-apm-api-keys
```

```suggestion
      ref: main
```
To be restored after ecommerce local pr merge